packages/govuk-frontend/src/govuk/components/summary-list/template.njk

```diff
@@ -1,6 +1,9 @@
-{%- macro _actionLink(action) %}
+{%- macro _actionLink(action, cardTitle = null) %}
```

Comment:
Can probably leave the Nunjucks `undefined` default?

Had a quick check and tend to use `!== undefined` elsewhere for optional things

Although any falsy value will work with `{%- if cardTitle %}`

```suggestion
{%- macro _actionLink(action, cardTitle) %}
```